### ISSUE TITLE
Allow an admin to override tasks' timeslot/date/time while duplicating/creating a delivery order

### ISSUE THREAD

**Issue by vladimir-8** [coopcycle/coopcycle-web#4613]
**In order to**  `allow creating order at the last minute`

**I want** `to allow an admin to override tasks' timeslot/date/time while duplicating/creating a delivery order`

**Whereas currently** `the admin is subject to the same validation rules as a client`

The platform will not allow duplicating an order with the same time slot if the time slot is no longer valid according to the validation rules.

**Additional context**

![Screenshot 2024-09-10 at 10 16 13](https://github.com/user-attachments/assets/7626f499-97d4-4aa5-a7f3-2804ff66eca9)

![Screenshot 2024-09-10 at 10 17 20](https://github.com/user-attachments/assets/9bf75d58-2d78-4152-99cf-ce96519bc996)

**Comment by HugoLCR** [comment]
Yes ! that's what I want

**Comment by Paul-Eraman-CoopCycle** [comment]
@Atala did you do this in the new orders form? i think you did

**Comment by Atala** [comment]
no i did not

the issue is that an admin should always have the "all the choices" date picker and not the timeslot picker ? @vladimir-8

**Comment by Paul-Eraman-CoopCycle** [comment]
No, more that ANY action that we "force" stores to do, we want to allow an admin to ignore. if an admin wants to program a pickup after a dropoff, they must have some reason, go for it. stores, its them not knowing what they are doing.

so if we could make ALL forced behavior non-obligatory for admins thatd be the goal

**Comment by Paul-Eraman-CoopCycle** [comment]
time estimate here @Atala @vladimir-8 ?

**Comment by Atala** [comment]
for the general conversation in needs to happen elsewhere, but keep in mind that validations and limitations are also here as a help for us to write code, code is based on assumptions. for ex that phone numbers are valid so you create a link so the person can call in app. that (maybe ^^) pickup is before dropoff as you dont handle this case in the pricing rule word - doesn't mean some assumptions/validations are bad / shouldn't be changed.

maybe 1 day work?

it is not very clear to me from the issue 
- what is the default selected datetime for the new order when duplicated? is it still the new timeslot?
- what do we present to admins? the timeslotS pickerS + a link to select the date OR just the general datetime picker directly?

**Comment by Paul-Eraman-CoopCycle** [comment]
Yeah i definitely dont know how that side of things works...
Generally i think it looks like we should make a design for this that starts with that side of things

**Comment by Atala** [comment]
- show timeslotS for admin
- add a tick button so the admin can switch to range picker

**Comment by vladimir-8** [comment]
Might be fixed by https://github.com/coopcycle/coopcycle-web/pull/4985 Needs testing
